pkg/state/manager.go

```diff
@@ -303,15 +380,17 @@ func (m *MManager) SaveKustomize(kustomize *Kustomize) error {
 
 // RemoveStateFile will attempt to remove the state file from disk
 func (m *MManager) RemoveStateFile() error {
-	statePath := m.V.GetString("state-file")
-	if statePath == "" {
-		statePath = constants.StatePath
-	}
+	// s, err := m.getStateSerializer()
+	// if err != nil {
+	// 	return errors.Wrap(err, "create state serializer")
+	// }
 
-	err := m.FS.RemoveAll(statePath)
-	if err != nil {
-		return errors.Wrap(err, "remove state file")
-	}
+	// err = s.Remove()
+	// if err != nil {
+	// 	return errors.Wrap(err, "remove state file")
+	// }
+
+	m.cachedState = &State{}
```

Comment:
Should this call the newly added `Remove` function too?

Also, commented code above

Reply:
good catch. this was just for debugging.